app/internal_packages/system-tray/lib/system-tray-icon-store.ts

```diff
@@ -84,24 +67,62 @@ class SystemTrayIconStore {
     this._updateIcon();
   };
 
+  // This implementation is windows only.
+  // On Mac the icon color is automatically inverted
+  // Linux ships with the icons used for a dark tray only
+  _dark = () => {
+    if (nativeTheme.shouldUseDarkColors && process.platform === 'win32') {
+      return "-dark";
+    }
+    return "";
```

Comment:
Ahh this is cool! It's kinda surprising that this isn't necessary on macOS too, but it looks like `isTemplateImg: true` makes the image auto-invert there.